cityofaustin/joplin
# Issues 3823, 3827: publish preflight

# Description
https://github.com/cityofaustin/techstack/issues/3823
https://github.com/cityofaustin/techstack/issues/3827

We've got IA publish requirements ready for fields, streamfields, and relations.







# Types of changes



# How can this be tested?



 

Go here: https://joplin-pr-3823-preflight-ia.herokuapp.com/admin

Try to publish a page without the publish requirements documented here: https://github.com/cityofaustin/techstack/issues/3823.

You will see error messages and not be redirected back to the home page.

# Checklist:
